### PR TITLE
adding hammer cli gutterball

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: hammer-cli-gutterball/rubygem-hammer_cli_gutterball.spec [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+%prep
+%setup -q -c -T
+mkdir -p .%{gem_dir}
+gem install --local --install-dir .%{gem_dir} \
+            --force %{SOURCE0}
+
+%install
+mkdir -p %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d
+install -m 755 %{SOURCE1} %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d/gutterball.yml
```
Comment (ehelms) [inline hammer-cli-gutterball/rubygem-hammer_cli_gutterball.spec]: You'll want to make this change first https://github.com/Katello/hammer-cli-katello/pull/305 (to the hammer-cli-gutterball repo)

Comment (jlsherrill) [inline hammer-cli-gutterball/rubygem-hammer_cli_gutterball.spec]: opened here:  https://github.com/Katello/hammer-cli-gutterball/pull/8/files

**Comment by jlsherrill** [comment]
[test]

**Comment by ehelms** [comment]
[test]

**Comment by ehelms** [comment]
ACK

**Pull request merged**